src/main/java/com/github/fishio/ICollisionArea.java

```diff
@@ -125,6 +145,14 @@ default boolean isReversed() {
 	double getSize();
 	
 	/**
+	 * Set the size of the collisionArea.
+	 * 
+	 * @param size
+	 * 		The new size.
+	 */
+	void setSize(double size);
```

Comment:
@MrCrime Please note that I simply moved this method upward, and did not add nor change it.

<review